src/api.rs

```diff
@@ -124,7 +124,7 @@ impl EncoderConfig {
   }
 }
 
-/// Contains all the speed settings
+/// Contain all the speed settings
```

Comment:
sometimes you use `contain` and sometimes you (or someone before you) use `contains` or `provides`: i think it's a redundant verb that could be dropped altogether